packages/xo-server/src/_pDebounceWithKey.spec.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-env jest */
+
+import { debounceWithKey, REMOVE_CACHE_ENTRY } from './_pDebounceWithKey'
+
+test('clear the cache of debounced function', async () => {
+  const key = 2
+
+  let i = 0
+  const debouncedFn = debounceWithKey(
+    function() {
+      return Promise.resolve(++i)
+    },
+    120e3,
```

Comment:
Why this value?

Either add a comment or use a duration which looks less specific.

Reply:
I choose a random value higher than the duration of the execution function it can also be `1e2`.